.github/workflows/aks-addon-tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     env:
       REGISTRY: ${{ needs.export-registry.outputs.registry }}
       E2E_IMG_TAG: "e2e-ci"
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Can you please add environment: e2e-tests so we can update the pull_request_target to pull_request?